src/features/todo/Element.tsx

```diff
@@ -2,20 +2,33 @@ import { EllipsisVerticalIcon } from '@heroicons/react/16/solid';
 import { Todo } from '../../types/Todo';
 import { Checkbox } from '../../components/Checkbox';
 import { DropdownMenu } from '../../components/DropdownMenu';
+import { dateFormatter } from '../../utils/dateFormatter';
 
 type Props = {
   item: Todo;
   onDeleteButtonClick: () => void;
+  onCheckboxClick: () => void;
 };
 
-export const Element = ({ item, onDeleteButtonClick }: Props) => {
-  const { name } = item;
+export const Element = ({ 
+  item,
+  onDeleteButtonClick,
+  onCheckboxClick,
+}: Props) => {
+  const { name, createdAt, done } = item;
+
+  const formattedDate = dateFormatter(createdAt);
 
   return (
     <div className="w-full h-12 bg-white rounded-lg flex justify-between items-center pr-3 relative overflow-hidden">
-      <div className="flex items-center h-full outline-none">
-        <Checkbox />
-        <h3 className="text-stone-700 font-medium">{name}</h3>
+      <div className="flex justify-between items-center h-full w-full outline-none px-1">
+        <div className="flex items-center gap-2">
+          <Checkbox checked={done} onCheckChange={onCheckboxClick} />
+          <h3 className="text-stone-700 font-medium">{name}</h3>
+        </div>
+        <span className="text-stone-400 justify-end text-xs">
```

Comment:
(Suggestion) What do you think about adjusting this a bit to the rest of the layout?
```suggestion
        <span className="text-stone-700 justify-end text-sm bg-stone-100 rounded-md h-8 flex items-center px-2 gap-x-1">
          <CalendarDaysIcon className="size-4" />
          {formattedDate}
        </span>
```
<img width="313" alt="Screenshot 2025-03-03 at 18 28 42" src="https://github.com/user-attachments/assets/5f8e69c5-5773-4145-a133-a1f93f4e3e19" />